corehq/apps/cleanup/tasks.py

```diff
@@ -160,13 +160,21 @@ def _is_monday():
     return datetime.utcnow().isoweekday() == 1
 
 
+def _get_first_doc_id(accessor, db_name):
+    doc_ids = accessor.get_doc_ids(db_name, limit=1)
+    try:
+        return next(doc_ids)
+    except StopIteration:
+        return None
```

Comment:
nit: since this is `limit=1`, you could also just wrap the generator in `list()`